documentation/book/proc-deploying-kafka-broker-kubernetes.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// assembly-kafka-broker.adoc
+
+[id='deploying-kafka-broker-kubernetes-{context}']
+= Deploying the Kafka broker to {KubernetesName}
+
+.Prerequisites
+
+* Before deploying a Kafka broker, the Cluster Operator must be deployed.
+* To deploy a Kafka cluster, a `Kafka` resource with the cluster configuration has to be created.
+
+.Procedure
+
+. If you are planning to use the Kafka broker for development or testing, create an ephemeral cluster
++
+[source,shell]
+----
+kubectl apply -f examples/kafka/kafka-ephemeral.yaml
+----
+
+. If you are planning to use the Kafka broker in production, create a persistent cluster
++
+[source,shell]
+----
+kubectl apply -f examples/kafka/kafka-persistent.yaml
+----
+
+.Additional resources
+* For more information about the deploying the Cluster Operator, see xref:assembly-cluster-operator.adoc[]
```

Comment:
"For more information about deploying..."? Not a native speaker but "the" sounds strange to me.

Reply:
You're 100% right!